sys/socket.txt

```diff
@@ -12,7 +12,6 @@ socket(domain flags[socket_domain], type flags[socket_type], proto int8) sock
 socketpair(domain flags[socket_domain], type flags[socket_type], proto int8, fds ptr[out, pipefd])
 accept(fd sock, peer ptr[out, sockaddr, opt], peerlen ptr[inout, len[peer, int32]]) sock
 accept4(fd sock, peer ptr[out, sockaddr, opt], peerlen ptr[inout, len[peer, int32]], flags flags[accept_flags]) sock
-# TODO: must not bind to port 0, that will result in a random port which is not reproducible
```

Comment:
This is not completely resolved unfortunately. There is a thing called autobind. It's normal to do connect without bind, then local port is chosen automatically by kernel. And it happens for passive sockets as well, you can do listen without bind, and then kernel will choose port (you can later obtain it with getsockname).
So please rephrase and move this TODO to, I don't know, listen, or maybe just to the top of the file.